modular/blocksyncer/database/bucket.go

```diff
@@ -113,3 +113,17 @@ func (db *DB) UpdateChargeSizeToSQL(ctx context.Context, objectID common.Hash, b
 	finalSql := fmt.Sprintf(sql, operation, tableName, tableName)
 	return finalSql, vars
 }
+
+// GetBucketByBucketName get bucket by bucket name
+func (db *DB) GetBucketByBucketName(ctx context.Context, bucketName string) (*bsdb.Bucket, error) {
```

Comment:
please import juno’s model In Blocksyncer

Reply:
fixed